packages/extension/src/permission/FirefoxPermission.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactElement } from 'react';
+import DailyDevLogo from '@dailydotdev/shared/src/components/Logo';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '@dailydotdev/shared/src/components/typography/Typography';
+import {
+  Button,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { VIcon } from '@dailydotdev/shared/src/components/icons';
+import { cloudinaryFeedBgLaptop } from '@dailydotdev/shared/src/lib/image';
+import { privacyPolicy } from '@dailydotdev/shared/src/lib/constants';
+import { FirefoxPermissionItem } from './FirefoxPermissionItem';
+import { FirefoxPermissionContainer } from './common';
+
+export enum FirefoxPermissionType {
+  Accepted = 'accepted',
+  Declined = 'declined',
+}
+
+interface FirefoxPermissionProps {
+  onUpdate(permission: FirefoxPermissionType): void;
+}
+
+export function FirefoxPermission({
+  onUpdate,
+}: FirefoxPermissionProps): ReactElement {
+  return (
+    <FirefoxPermissionContainer>
+      <img
+        src={cloudinaryFeedBgLaptop}
+        alt="a glowing background"
+        className="pointer-events-none absolute -top-10 -z-1 select-none"
+      />
+      <span className="py-6">
+        <DailyDevLogo />
+      </span>
+      <div className="flex w-full max-w-[33rem] flex-col gap-2 rounded-10 bg-surface-float p-4">
+        <Typography tag={TypographyTag.H1} type={TypographyType.Title3} bold>
+          Additional permissions needed
+        </Typography>
+        <Typography
+          type={TypographyType.Callout}
+          color={TypographyColor.Tertiary}
+        >
+          Ready to go? 🟢 Give us the green signal by accepting the below
+          permissions. Enhance your daily.dev experience with personalized
+          content and connect with fellow developers.
+        </Typography>
+        <FirefoxPermissionItem
+          className="mt-2"
+          title="Ads pixel tracking"
+          description="Enable pixel tracking for fewer, non-intrusive ads, helping us in keeping the experience free and relevant."
```

Comment:
Maybe raise with product if they don't want to add people can also buy plus and no ads are shown here?

Reply:
Good point. Let me check.